1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,18 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  let start = 0;
+  let toxicList = [];
+  while (toxicList.length < n) {
+    if (start % 2 === 0) {
+      toxicList.push(start);
+    }
+
+    start++;
+  }
+  return result.toString();
```

Comment:
You function could not run as it called a variable called **result** which is not defined.